superset/assets/src/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -464,6 +466,15 @@ export default class AnnotationLayer extends React.PureComponent {
                   }
                 }}
               />
+              <TextControl
+                hovered
+                name="annotation-override-param"
+                label="Override params"
+                description={'Override the params in the chart'}
+                placeholder=""
+                value={overrides.params}
+                onChange={v => this.setState({ overrides: { ...overrides, params: v } })}
```

Comment:
Can you use a more descriptive variable name than `v`

Reply:
Ah just noticed that this is how it's handled in other places in the file. I'm not a fan of the choice on their end, but I suppose we should maintain consistency.